DNN Platform/DotNetNuke.DependencyInjection/Extensions/TypeExtensions.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using System.Reflection;
+
+namespace DotNetNuke.DependencyInjection.Extensions
+{
+    /// <summary>
+    /// <see cref="Type"/> specific extneions to be used
```

Comment:
Minor typo in the comment. "extnesions" should be "extensions"

Reply:
Good find, thanks for reporting the typo.